docker/lisk-build/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:trusty
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install --no-install-recommends \
+        curl sudo unzip wget zip jq moreutils python \
+        autoconf automake build-essential git libpq-dev libtool libncurses5-dev libreadline6-dev \
+        chrpath zlib1g-dev libssl-dev ca-certificates && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 6.14.2
+ENV NVM_VERSION 0.33.11
+ENV NVM_INSTALL_CHECKSUM bb7e8f51023ab8b054c6578591fa0dc361ceeb08744e5cd1f0e551235c4912b2
+
+RUN mkdir -p $NVM_DIR && \
+    wget -O /tmp/install_nvm.sh https://raw.githubusercontent.com/creationix/nvm/v$NVM_VERSION/install.sh && \
+    echo "${NVM_INSTALL_CHECKSUM} /tmp/install_nvm.sh" | sha256sum --check && \
+    bash /tmp/install_nvm.sh
+
+
+# install node and npm
+RUN /bin/bash -c "source $NVM_DIR/nvm.sh && nvm install $NODE_VERSION && nvm alias default $NODE_VERSION && nvm use default"
```

Comment:
Split this line so that it's not so long; maybe merge it with the `RUN` just before it also?